### PR TITLE
Add new visual and accessibility tests

### PR DESCRIPTION
**Pull request by jorge-ramirez-arredondo**
[US127919](https://rally1.rallydev.com/#/431372673576d/iterationstatus?detail=%2Fuserstory%2F603444721280&fdp=true?fdp=true)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,13 +10,16 @@
...
+    "test:headless:watch": "karma start --auto-watch=true --single-run=false",
+    "test:visual": "mocha './test/**/*.visual-diff.js' -t 10000 --require esm",
+    "test:visual:golden": "mocha './test/**/*.visual-diff.js' -t 10000 --golden --require esm"
   },
   "author": "D2L Corporation",
   "license": "Apache-2.0",
   "devDependencies": {
     "@babel/eslint-parser": "^7",
     "@brightspace-ui/stylelint-config": "^0.1",
+    "@brightspace-ui/visual-diff": "^6.0.1",
```
Comment (dlockhart) [inline package.json:22]: If you can get away with _not_ including `visual-diff`, `esm` and `puppeteer` as devDependencies, things will go a lot smoother in the future.

Good or bad, the way this works today is the [visual-diff GitHub Action installs these](https://github.com/BrightspaceUI/actions/blob/master/visual-diff/action.yml#L33). And when a new version of puppeteer or our visual-diff library comes out, we update the action and everyone just magically gets that update. But if you're also including these as dependencies, the versions will get out of sync and you'll be testing something different locally vs. in the action.

File: test/.eslintrc.json [hunk 2]
```diff
@@ -1,3 +1,4 @@
 {
-	"extends": "brightspace/open-wc-testing-config"
+	"extends": "brightspace/open-wc-testing-config",
+	"ignorePatterns": ["screenshots/current/*", "screenshots/golden/*"]
```
Comment (dlockhart) [inline test/.eslintrc.json:3]: I'm surprised you needed to do this as I've never seen this... if these go in the `.gitignore` instead maybe you can avoid duplicating them here?

File: package.json (hunk 1 above)
Comment (jorge-ramirez-arredondo) [reply package.json:22]: Ah, okay. Is the expectation that you install these in your machine globally instead?
The [testing locally section in the visual-diff README](https://github.com/BrightspaceUI/visual-diff#locally) didn't say anything about where these libraries should be coming from.

File: test/.eslintrc.json (hunk 2 above)
Comment (jorge-ramirez-arredondo) [reply test/.eslintrc.json:3]: These are in the `.gitignore` already, but I was getting linting errors from the generated visual-diff report html file. Not sure if there is a better way to address this.

File: package.json (hunk 1 above)
Comment (dlockhart) [reply package.json:22]: Yeah exactly. The visual-diff step from the `npm init` generator would add [this section](https://github.com/BrightspaceUI/create/blob/main/src/generators/test-visual-diff/templates/_README.md) to your README when you use it, but I think you were adding visual-diff after the fact so maybe didn't get it.

Comment (jorge-ramirez-arredondo) [reply package.json:22]: Oh, lol, it's there. I didn't think to check in the project README, I went straight to the visual-diff README.
I'll remove these then. 👍

File: test/.eslintrc.json (hunk 2 above)
Comment (dlockhart) [reply test/.eslintrc.json:3]: Hmm ok maybe that does sound familiar 🤔

**Review by dlockhart — APPROVED** [review]

**Pull request merged**